packages/surveys/src/index.spec.ts

```diff
@@ -0,0 +1,82 @@
+import SurveyPlugin from "@jspsych/plugin-survey";
+import { initJsPsych } from "jspsych";
+import { Trial as ConsentTrial } from "./consent";
+import { Trial as ExitTrial } from "./exit";
+import Surveys from "./index";
+import { consentSurveyFunction, exitSurveyFunction } from "./utils";
+
+jest.mock("@jspsych/plugin-survey");
+jest.mock("jspsych");
+jest.mock("./utils");
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+test("Consent Survey", () => {
+  Object.defineProperty(global, "document", {
+    value: {
+      addEventListener: jest.fn(),
+      querySelector: jest.fn().mockReturnValue({ style: { display: "" } }),
+    },
+  });
+
+  const jsPsych = initJsPsych();
+  const consent = new Surveys.consent(jsPsych);
+  const display_element = jest.fn() as unknown as HTMLElement;
+  const trialInfo = { survey_function: jest.fn() } as unknown as ConsentTrial;
+
+  consent.trial(display_element, trialInfo);
+
+  expect(SurveyPlugin.prototype.trial).toHaveBeenCalledWith(display_element, {
+    ...trialInfo,
+    survey_function: consentSurveyFunction(trialInfo.survey_function),
+  });
+  expect(SurveyPlugin.prototype.trial).toHaveBeenCalledTimes(1);
+});
+
+test("Exit Survey", () => {
+  Object.defineProperty(global, "window", {
+    value: {
+      chs: { study: { attributes: { contact_info: jest.fn() } } },
+    },
+  });
+
+  const exit = new Surveys.exit(initJsPsych());
+  const display_element = jest.fn() as unknown as HTMLElement;
+  const trialInfo = {
+    survey_function: jest.fn(),
+    survey_json: jest.fn(),
+  } as unknown as ExitTrial;
+
+  exit.trial(display_element, trialInfo);
+
+  expect(SurveyPlugin.prototype.trial).toHaveBeenCalledTimes(1);
+  expect(SurveyPlugin.prototype.trial).toHaveBeenCalledWith(display_element, {
+    ...trialInfo,
+    survey_function: exitSurveyFunction,
+    survey_json: Surveys.exit.prototype["surveyParameters"](trialInfo),
```

Comment:
We don't need to do this now, but at some point in the future we should think about testing the data that comes out of our plugins too. E.g. simulate some responses and check that the expected values show up in the trial data. 
We can also check for any special response properties that should be set via these plugins: `withdrawal` from the exit survey plugin, and `survey_consent` and `completed_consent_frame` from the consent survey plugin.

Reply:
Ok I agree, that makes sense. 
How about adding tests just for any data that we're adding, e.g. custom CHS IDs? (We haven't implemented that yet, but when we do).